tcf_website/static/reviews/review.css

```diff
@@ -1,7 +1,21 @@
-.review .review-header {
-    color: var(--accent-color);
-    font-size: 1.25rem;
-}  
+.review-content .review-text.collapse:not(.show) {
+  display: block;
+  height: 12rem;
```

Comment:
If possible I would adjust these heights for mobile.
The reviews are still too tall on small displays
![image](https://user-images.githubusercontent.com/45245651/107434459-d0a77080-6af8-11eb-9431-6f4a00eb0d17.png)

Reply:
Ok I'm a little confused actually because the collapsed reviews are a good height on mobile but it seems like the threshold for collapsing is not the same as that height so some reviews that are too tall aren't collapsable
http://127.0.0.1:8000/course/598/7285
![image](https://user-images.githubusercontent.com/45245651/107434749-2aa83600-6af9-11eb-9629-b488357e9cdc.png)

Reply:
I can't seem to reproduce this issue...let's chat in discord